pyproject.toml

```diff
@@ -20,7 +20,8 @@ dependencies = [
     "requests",
     "aind-codeocean-api>=0.4.0",
     "pydantic>=2.0",
-    "pydantic-settings>=2.0"
+    "pydantic-settings>=2.0",
+    "aind-data-schema",
```

Comment:
suggestion: move `aind-data-schema` dependency to optional `[helpers]` to keep things lightweight. @jtyoung84 what do you think?

Reply:
We discussed and settled on putting it in the main dependencies